moped-editor/src/views/projects/ProjectsTable.js

```diff
@@ -9,27 +9,57 @@ import {
     TableCell,
     TableHead,
     TableRow,
-    makeStyles, Link,
+    makeStyles, Link, Chip, Icon,
```

Comment:
weird formatting happening here. I mentioned this in another PR but can you confirm that your ES Linter + Prettier is configured to run on file save Sergio?